runtime/gc_include/j9modron.h

```diff
@@ -145,6 +145,7 @@ typedef jvmtiIterationControl J9MODRON_REFERENCE_CHAIN_WALKER_CALLBACK(J9Object
 #define J9GC_REFERENCE_TYPE_CLASS_ARRAY_CLASS -12 /**< reference to an array class' component type class */
 #define J9GC_REFERENCE_TYPE_CLASS_NAME_STRING -13 /**< reference to a class' name string */
 #define J9GC_REFERENCE_TYPE_CALL_SITE -14 /**< call site reference to an object */
+#define J9GC_REFERENCE_TYPE_FCC -15 /**< reference to a class in flattened class cache */
```

Comment:
I would keep explicit notion this is class in the name `J9GC_REFERENCE_TYPE_CLASS_FCC`

Reply:
Does `J9GC_REFERENCE_TYPE_FCC_CLASS` work fine?